From 17d0e3dfcd0690df0e7b212fedcb95402f16935d Mon Sep 17 00:00:00 2001
From: Jose Dapena Paz <jdapena@igalia.com>
Date: Fri, 23 Jul 2021 10:17:49 +0000
Subject: [PATCH] IWYU: missing include for using std::vector in hash password
 manager.
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Fix build breakage:
../../components/password_manager/core/browser/hash_password_manager.h:44:8: error: ‘vector’ in namespace ‘std’ does not name a template type
   44 |   std::vector<PasswordHashData> RetrieveAllPasswordHashes();
      |        ^~~~~~

Bug: 819294
Change-Id: I8c8a4ec3972eedb87a312c5ec56adf4a21b1b2a2
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/3041046
Commit-Queue: Vasilii Sukhanov <vasilii@chromium.org>
Reviewed-by: Vasilii Sukhanov <vasilii@chromium.org>
Cr-Commit-Position: refs/heads/master@{#904696}
---
 components/password_manager/core/browser/hash_password_manager.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/components/password_manager/core/browser/hash_password_manager.h b/components/password_manager/core/browser/hash_password_manager.h
index c762c5a8c7713..85e656edcba11 100644
--- a/components/password_manager/core/browser/hash_password_manager.h
+++ b/components/password_manager/core/browser/hash_password_manager.h
@@ -6,6 +6,7 @@
 #define COMPONENTS_PASSWORD_MANAGER_CORE_BROWSER_HASH_PASSWORD_MANAGER_H_
 
 #include <string>
+#include <vector>
 
 #include "base/callback.h"
 #include "base/callback_list.h"
